ekncontent/ekncontent/eknc-query-object.c

```diff
@@ -827,6 +866,27 @@ get_corrected_query (EkncQueryObject *self,
   return g_steal_pointer (&query_clause);
 }
 
+/*
```

Comment:
For future reference, static functions don't need this level of detail in the documentation, but it's already here, so best to leave it.

Reply:
Yeah. I was mostly just copying the style I saw in `get_filter_clause`, etc